scripts/ci/rerun-integration-tests.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 
-set -e
```

Comment:
not sure if I like 🤔
wasnt it you telling me `set -e` is basicly mandatory? 😅

Reply:
I re-set it, later on before the test runs start